packages/wrangler/src/pages/deploy.tsx

```diff
@@ -73,38 +74,49 @@ export function Options(yargs: CommonYargsArgv) {
 		});
 }
 
-export const Handler = async ({
-	_,
-	directory,
-	projectName,
-	branch,
-	commitHash,
-	commitMessage,
-	commitDirty,
-	skipCaching,
-	bundle,
-	noBundle,
-	config: wranglerConfig,
-}: PublishArgs) => {
+export const Handler = async (args: PublishArgs) => {
+	let { branch, commitHash, commitMessage, commitDirty } = args;
+
 	// Check for deprecated `wrangler pages publish` command
-	if (_[1] === "publish") {
+	if (args._[1] === "publish") {
 		logger.warn(
 			"`wrangler pages publish` is deprecated and will be removed in the next major version.\nPlease use `wrangler pages deploy` instead, which accepts exactly the same arguments."
 		);
 	}
 
-	if (wranglerConfig) {
-		throw new FatalError("Pages does not support wrangler.toml", 1);
+	if (args.config) {
+		throw new FatalError(
```

Comment:
@penalosa just dbl checking whether we indeed want to fail here

Reply:
Yeah 👍